x/capability/capability_test.go

```diff
@@ -70,12 +70,15 @@ func (suite *CapabilityTestSuite) TestInitializeMemStore() {
 
 	// Mock app beginblock and ensure that no gas has been consumed and memstore is initialized
 	ctx = suite.app.BaseApp.NewContext(false, cmtproto.Header{}).WithBlockGasMeter(storetypes.NewGasMeter(50))
-	prevGas := ctx.BlockGasMeter().GasConsumed()
+	prevBlockGas := ctx.BlockGasMeter().GasConsumed()
+	prevGas := ctx.GasMeter().GasConsumed()
 	restartedModule := capability.NewAppModule(suite.cdc, *newKeeper, true)
 	restartedModule.BeginBlock(ctx, abci.RequestBeginBlock{})
+	gasUsed := ctx.GasMeter().GasConsumed()
 	suite.Require().True(newKeeper.IsInitialized(ctx), "memstore initialized flag not set")
-	gasUsed := ctx.BlockGasMeter().GasConsumed()
+	blockGasUsed := ctx.BlockGasMeter().GasConsumed()
 
+	suite.Require().Equal(prevBlockGas, blockGasUsed, "beginblocker consumed block gas during execution")
```

Comment:
```suggestion
	suite.Require().Equal(prevBlockGas, blockGasUsed, "ensure beginblocker consumed no block gas during execution")
```
can you update the below check's description as well.